src/colorizer/ColorizeCanvas.ts

```diff
@@ -195,23 +197,6 @@ export default class ColorizeCanvas {
     this.getCurrentFrame = this.getCurrentFrame.bind(this);
     this.setOutOfRangeDrawMode = this.setOutOfRangeDrawMode.bind(this);
     this.updateScaling = this.updateScaling.bind(this);
-
-    // Set up the canvas overlay as a sibling in the DOM layout
-    // by creating a dummy parent container.
-    this.canvasContainer = document.createElement("div");
-    this.canvasContainer.appendChild(this.renderer.domElement);
-    this.canvasContainer.appendChild(this.overlay.domElement);
-    this.canvasContainer.style.position = "relative";
-    this.overlay.domElement.style.position = "absolute";
-    this.overlay.domElement.style.left = "0";
-    this.overlay.domElement.style.top = "0";
-  }
-
-  /**
-   * The DOM element containing the canvas and its overlay.
-   */
-  get domElement(): HTMLDivElement {
-    return this.canvasContainer;
   }
 
   get canvasElement(): HTMLCanvasElement {
```

Comment:
might consider moving back to calling this `domElement`, for consistency with the property on `renderer` that it's actually getting